ec2Bootstrap/nightly.sh

```diff
@@ -17,7 +17,7 @@ NIGHTLY="--nightly date=$DATE_START,weeksBack=$WEEKS"
 
 echo "Executing nightly script with date: $DATE_START and weeks: $WEEKS"
 
-CMD="sbt -J-Xmx30G -J-Xms30G \"; project client; run-main com.broilogabriel.Client --sourceCluster=$SOURCE_CLUSTER --targetCluster=$TARGET_CLUSTER --sources=$SOURCE_IPS --targets=$TARGET_IPS $NIGHTLY $INDEX\""
+CMD="sbt -J-Xmx25G -J-Xms25G \"; project client; run-main com.broilogabriel.Client --sourceCluster=$SOURCE_CLUSTER --targetCluster=$TARGET_CLUSTER --sources=$SOURCE_IPS --targets=$TARGET_IPS $NIGHTLY $INDEX\""
```

Comment:
It's possible to replace: `sbt -J-Xmx25G -J-Xms25G \"; project client; run-main com.broilogabriel.Client `
By: `sbt -J-Xmx25G -J-Xms25G \"project client\" \"run`